src/main/java/gui/misc/TweakingHelper.java

```diff
@@ -36,31 +52,105 @@
             }
     };
 
-    public static final Color COLOR_PRIMARY = colors[colorChoice][0]; // main (light) color
-    public static final Color COLOR_SECONDARY = colors[colorChoice][1]; // dark color
-    public static final Color COLOR_TERTIARY = colors[colorChoice][2]; // very dark color
-    public static final Color COLOR_QUADRATORY = colors[colorChoice][3]; // very light color
+    @Getter
+    private static final String[] colorNames = {
+        "blue",
+        "orange",
+        "green",
+        "red"
+    };
+
+    /**
+     * @return amount of colors currently defined.
+     */
+    public static int getNumberOfColors() {
+        return colors.length;
+    }
 
     /*
      * Some whites and grays used throughout the application.
      */
 
-    public static final Color COLOR_BACKGROUND = Color.WHITE; // main bg color
-    public static final Color COLOR_BACKGROUND_HIGH =
-            Color.rgb(245, 245, 245); // heightened bg color
+    @Getter
+    private static Color backgroundColor = Color.WHITE; // main bg color
+    @Getter
+    private static Color backgroundHighColor = Color.rgb(245, 245, 245); // heightened bg color
+    @Getter
+    private static Color backgroundContrastColor = Color.WHITE; // Color.BLACK;
 
-    // string versions of main colors
-    public static final String STRING_PRIMARY = getStringFromColor(COLOR_PRIMARY);
-    public static final String STRING_SECONDARY = getStringFromColor(COLOR_SECONDARY);
-    public static final String STRING_TERTIARY = getStringFromColor(COLOR_TERTIARY);
-    public static final String STRING_QUADRATORY = getStringFromColor(COLOR_QUADRATORY);
-    public static final String STRING_BACKGROUND = getStringFromColor(COLOR_BACKGROUND);
-    public static final String STRING_BACKGROUND_HIGH = getStringFromColor(COLOR_BACKGROUND_HIGH);
+    /*
+     * Getters
+     */
 
-    // general layout variables used throughout application.
-    public static final int GENERAL_SIZE = 10000;
-    public static final int GENERAL_SPACING = 10;
-    public static final int GENERAL_PADDING = 20;
+    /**
+     * @return primary color.
+     */
+    public static Color getPrimaryColor() {
```

Comment:
Could you refactor this into something like getNthColor(int n) { return colors[colorChoice][n-1];}?
This is a lot of pretty similar code